src/container/srv_container.c

```diff
@@ -5696,30 +5715,32 @@ ds_cont_svc_set_prop(uuid_t pool_uuid, uuid_t cont_uuid,
 
 rechoose:
 	ep.ep_grp = NULL; /* primary group */
-	rc = rsvc_client_choose(&client, &ep);
+	rc        = rsvc_client_choose(&client, &ep);
 	if (rc != 0) {
-		D_ERROR(DF_CONT": cannot find pool service: "DF_RC"\n",
-			DP_CONT(pool_uuid, cont_uuid), DP_RC(rc));
+		D_ERROR(DF_UUID ": cannot find pool service: " DF_RC "\n", DP_UUID(pool_uuid),
+			DP_RC(rc));
```

Comment:
minor: The `DL_ERROR` macro is preferred now.
```suggestion
		DL_ERROR(rc, DF_UUID ": cannot find pool service", DP_UUID(pool_uuid));
```

Reply:
Fixed